### ISSUE TITLE
Updating an existing plugin fails with missing license message although license is present

### ISSUE THREAD

**Issue by burkmarr** [qgis/QGIS-Website#386]
### URL

https://plugins.qgis.org/plugins/add/

### On what type of device(s) did you see this bug?

Desktop

### On which browser(s) are you seeing the problem?

Chrome

### Summary

Updating an existing plugin, which I've done many times before, fails with the message shown below:

**There were errors reading plugin package (please check also your plugin's metadata). Cannot find LICENSE in the plugin package. This file is required, please consider adding it to the plugin package.**

I already had a license in the package called license.txt. I tried renaming that to LICENSE.txt but with the same result. I've also confirmed that I have all mandatory metadata.txt fields (as described here: https://docs.qgis.org/3.34/en/docs/pyqgis_developer_cookbook/plugins/plugins.html#metadata-txt).

I noticed that this problem has also been noted by by someone else recently on Stackexchange (https://gis.stackexchange.com/questions/484049/cannot-find-license-in-the-plugin-package-error-when-updating-qgis-plugin/).

![image](https://github.com/user-attachments/assets/b13ab6c2-5bc5-445e-905c-6c098e7ea54c)

**Comment by timlinux** [comment]
Thank you for your report - can you please move it to the plugin repo.

https://github.com/qgis/QGIS-Django/issues

Out of curiosity, what workflow did you follow to end up filing your ticket here (so we can try to fix the source of the issue of reporting in the wrong repo)?

Thanks

Tim

**Comment by burkmarr** [comment]
I found this: https://www.qgis.org/resources/support/bug-reporting/ and looked under 'Where to report'. My problem wasn't with QGIS desktop, server or APIs. Website seemed the best fit since I was trying to use a tool from the plugin subdomain of the website.

I've managed to fix my problem now. I think there is a new requirement that the license is named LICENSE without any file extension. But this doesn't appear to be documented - I found a clue on a QGIS pull request. I'll raise the issue on https://github.com/qgis/QGIS-Documentation/issues.
